proposals/scopes.md

```diff
@@ -113,13 +113,13 @@ The following information describes a scope in the source map:
   - a javascript expression that can be evaluated by the debugger in the corresponding generated scope to get the binding's value (if such an expression is available)
 
 The following code snippet specifies the scope information **conceptually** in TypeScript notation. See the [Encoding](#encoding) section
-on how this information is actually VLQ encoded.
+on how this information is actually VLQ encoded. We chose the name `GeneratedRange` instead of `GeneratedScope` to make it explicit that a `GeneratedRange` does not necessarily correspond to a lexical ECMAScript scope (e.g. in the case of an inlined function body).
 
 ```ts
 interface SourceMap {
   // ...
   originalScopes?: OriginalScope[];
-  generatedScopes?: GeneratedScope;
+  generatedRanges?: GeneratedRange;
```

Comment:
Should this be `generatedRange` singular if it only takes a single range?

Reply:
`GeneratedRange` contains child ranges.

Reply:
I can see how both names fit. I'll land as is but we can bike-shed the names in the future :)